package/MDAnalysis/coordinates/GRO.py

```diff
@@ -237,29 +240,31 @@ class GROWriter(base.WriterBase):
     }
     fmt['xyz_v'] = fmt['xyz'][:-1] + "{vel[0]:8.4f}{vel[1]:8.4f}{vel[2]:8.4f}\n"
 
-    def __init__(self, filename, convert_units=None, **kwargs):
+    def __init__(self, filename, convert_units=None, n_atoms=None, **kwargs):
         """Set up a GROWriter with a precision of 3 decimal places.
 
-        :Arguments:
-           *filename*
-              output filename
+        Parameters
+        -----------
+        filename : str
+            output filename
+
+        n_atoms : int (optional)
+            number of atoms
+
         """
         self.filename = util.filename(filename, ext='gro')
+        self.n_atoms = n_atoms
 
         if convert_units is None:
             convert_units = flags['convert_lengths']
         self.convert_units = convert_units  # convert length and time to base units
 
-    def write(self, selection, frame=None):
+    def write(self, obj):
         """Write selection at current trajectory frame to file.
 
-        :Arguments:
-          selection
-              MDAnalysis AtomGroup (selection or Universe.atoms)
-              or also Universe
-        :Keywords:
-          frame
-              optionally move to frame number *frame*
+        Parameters
+        -----------
+        obj : AtomGroup / Universe / Timestep
```

Comment:
You still need to add new version changed strings below. Also after the first Paramter/Returns only descriptions with headings should be in the doc string